src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffendersResourceTest.java

```diff
@@ -589,6 +590,95 @@ public void testRecallAPrisoner() {
         assertThat(releaseResponse.getStatusCodeValue()).isEqualTo(200);
     }
 
+    @Test
+    public void testReceiveAPrisonerAsNewBooking() {
+        final var token = authTokenHelper.getToken(AuthToken.CREATE_BOOKING_USER);
+
+        final var body = Map.of("receivedPrisonId", "SYI", "fromLocationId", "COURT1", "movementReasonCode", "24", "youthOffender", "true", "imprisonmentStatus", "CUR_ORA", "cellLocation", "SYI-A-1-1");
+
+        final var newBookingEntity = createHttpEntity(token, body);
+
+        final var response =  testRestTemplate.exchange(
+            "/api/offenders/{nomsId}/new-booking",
+            POST,
+            newBookingEntity,
+            new ParameterizedTypeReference<String>() {
+            },
+            "Z0022ZZ"
+        );
+        assertThat(response.getStatusCodeValue()).isEqualTo(200);
+
+        final var searchToken  = authTokenHelper.getToken(AuthToken.GLOBAL_SEARCH);
+        final var httpEntity = createHttpEntity(searchToken, format("{ \"offenderNos\": [ \"%s\" ] }", "Z0022ZZ"));
+
+        final var searchResponse = testRestTemplate.exchange(
+            "/api/prisoners",
+            POST,
+            httpEntity,
+            new ParameterizedTypeReference<String>() {
+            });
+
+        assertThatJsonFileAndStatus(searchResponse, 200, "new_booking_prisoner.json");
+
+        final var releaseBody = createHttpEntity(token, Map.of("movementReasonCode", "CR", "commentText", "released prisoner today"));
```

Comment:
maybe also do a GET to prove the booking is really there

Reply:
Yes will add in next PR